cloudos/__main__.py

```diff
@@ -255,11 +262,11 @@ def run(apikey,
         disable_ssl_verification,
         ssl_cert):
     """Submit a job to CloudOS."""
-    print('Executing run...')
+    cloudos_url = cloudos_url.rstrip('/')
```

Comment:
Instead of stripping extra slashes in each function, how about we handle this in `Cloudos` class itself? 
This will centralise the handling

Reply:
@s0rthak that's actually a great suggestion and it was also my first attempt of implementation. I implemented this setter here https://github.com/lifebit-ai/cloudos-cli/pull/171/commits/0add0ff5d3d96a7cb9f722fe939c38097921ca56 but it didn't work. The issue is that `cloudos_url` is the first parameter of the class and we cannot have setters in the first and not having default values for the following. Sure, I could change order in `Cloudos` class, but the issue is `Job`  and `Queue` classes that are inheriting from `Cloudos` . At that point, it required more modification than just the stripping at each function, so I went for the simpler solution. What do you think? Do you think it's still worth to change the class variables for the three classes?

Reply:
Let's get this PR out of the way and we can improve `cloudos_url` handling in a follow-up, no worries!